app/src/main/java/org/mozilla/fenix/components/metrics/MetricsStorage.kt

```diff
@@ -150,13 +163,8 @@ internal class DefaultMetricsStorage(
          * - tracking is still enabled through Nimbus
```

Comment:
should comment ` * - user is within 2-28 days of install` be removed now?

Reply:
Yes, thank you!